### PR TITLE
Ensure PagerHeaderToolsGroup visibility mods are optional

### PR DESCRIPTION
**Pull request by DarrienG**
**What**: 

In the official demos here:
https://www.patternfly.org/v4/documentation/react/demos/pagelayout

You can see from the horizontal nav example that only the default and lg
props are specified.

This is legal in JS and I assume is how it is expected to work. Set a
default and add exceptions for other sizes you'd like for other
properties.

In ts, there is a hard requirement on setting each of the properties
because it is typechecked unlike JS which is super tedious (especially
if you just want something to always be visible).

This brings the convenience of the JS examples to TS as well.

I don't think there are any issues for this.

**Additional issues**:

No

### CODE REVIEW

**Comment by DarrienG** [comment]
Added issue here: https://github.com/patternfly/patternfly-react/issues/4394

File: packages/react-core/src/components/Page/PageHeaderToolsGroup.tsx [hunk 1]
```diff
@@ -11,11 +11,11 @@ export interface PageHeaderToolsGroupProps extends React.HTMLProps<HTMLDivElemen
   /** Visibility at various breakpoints. */
   visibility?: {
     default: 'hidden' | 'visible';
```
Comment (redallen) [inline packages/react-core/src/components/Page/PageHeaderToolsGroup.tsx:13]: This should be optional too, no?

Comment (DarrienG) [reply packages/react-core/src/components/Page/PageHeaderToolsGroup.tsx:13]: I figured the intent was that default is supposed to be enforced and exceptions specified, but now looking at how some of the other components work:

      <Flex direction={{}}>

Seems like default is generally optional. Let me update real quick...

**Comment by DarrienG** [comment]
Yeah of course! It's a great library and I appreciate the work you folks have done on it as well.

Last commit should fix review comments.

**Pull request merged**